dgraphtest/compose_cluster.go

```diff
@@ -28,11 +28,23 @@ func NewComposeCluster() *ComposeCluster {
 }
 
 func (c *ComposeCluster) Client() (*GrpcClient, func(), error) {
-	return nil, nil, errNotImplemented
+	client, err := testutil.DgraphClient(testutil.SockAddr)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	cleanup := func() {
```

Comment:
this could be a one liner, you could also just return directly without assignment